apr/tests/test.bats

```diff
@@ -0,0 +1,24 @@
+@test "package directory for package ident ${TEST_PKG_IDENT} exists" {
+  [ -d "/hab/pkgs/${TEST_PKG_IDENT}" ]
+}
+
+expected_version="$(echo $TEST_PKG_IDENT | cut -d/ -f 3)"
+@test "apr exe runs" {
+  run hab pkg exec $TEST_PKG_IDENT apr-1-config --version
+  [ $status -eq 0 ]
+}
+
+@test "apr exe output mentions expected version $expected_version" {
+  run hab pkg exec $TEST_PKG_IDENT apr-1-config --version
+  [[ "$output" =~ $expected_version ]]
+}
+
+@test "apr exe runs" {
```

Comment:
This test and the one at line 21 seem to be duplicates of the tests at lines 5 and 11

Reply:
Removed additional tests from original copy/pasting effort! Also updated one to change what it runs to I'm not using the version flag in both tests.